### PR TITLE
add scroll to settings sidebar

### LINKED ISSUE #47: Make settings/controls scrollable
If users have small screens or use scaling, then the settings might be so long that they continue beyond the screen. There is currently no way for the user to access those settings.

It should be relatively easy to implement a scrollbar to allows the user to scroll from the top settings to the bottom settings.

image

### PR DESCRIPTION
**Pull request by lewisyng**
Fixes #47

### CODE REVIEW

**Comment by atjn** [comment]
A quick tip: if you write "Fixes #47" in the description of your pull request, then it will be linked back to the issue I created, and will automagically close the issue when we merge this.

File: source/style.css [hunk 1]
```diff
@@ -473,3 +472,8 @@ body.visualization-in-focus #vertical-align {
 #controls-pane.open > .toggle:hover > svg {
 	transform: rotate(210deg);
 }
+
+#controls-pane #controls {
+	overflow-y: scroll;
```
Comment (atjn) [inline source/style.css]:
```suggestion
	overflow-y: auto;
```
This will hide the scrollbar when the user can't scroll. I think that looks prettier :)

**Comment by lewisyng** [comment]
Thanks so much for the advice. I fixed the css and implemented your commit message tip :)

**Comment by atjn** [comment]
Thank you for your work :)

**Pull request merged**